### DISCUSSION TITLE
Backend Performance Testing

### DISCUSSION THREAD

**Discussion by ASaiAnudeep** [test-results-reporter/reporter#56] [Ideas]
In general, performance tests reporting provides an overall result, test or result analysis and recommendations to the dev team on the applications performance perspective. Analysis of the results, identifying the bottle necks and system issues would mostly a manual effort.

Our target is to provide a high level insights on the overall test results by allowing the users to customise the information they wanted to see in their communication platforms.

#### Sample Information 

- Total Requests
- Total Duration
- Requests/Second
- Error Rate
- Failure Rate
- Latency
- Average Response Time
- 99 Percentile

The same list of metrics would apply at transaction level as. A transaction could be a simple HTTP request or flow of requests.

### Next Stage

Determining whether the overall performance results are satisfactory or not is an important factor. Like a **pass** or **fail**. This could be fetched from either from the performance tool or this tool should provide the thresholds and determine the result.

**Comment by Sudeep05** [comment]
JMeter - [aggregate.csv](https://github.com/test-results-reporter/reporter/files/9159902/aggregate.csv)
